docs/notebooks/atmodel_tutorial.ipynb

```diff
@@ -0,0 +1,1061 @@
+     "text": [
+      "Number of authors: 2479\n",
+      "Number of unique tokens: 6996\n",
+      "Number of documents: 1740\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Number of authors: %d' % len(author2doc))\n",
+    "print('Number of unique tokens: %d' % len(dictionary))\n",
+    "print('Number of documents: %d' % len(corpus))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Train and use model\n",
+    "\n",
+    "We train the author-topic model on the data prepared in the previous sections. \n",
+    "\n",
+    "The interface to the author-topic model is very similar to that of LDA in Gensim. In addition to a corpus, ID to word mapping (`id2word`) and number of topics (`num_topics`), the author-topic model requires either a author to document ID mapping (`author2doc`), or the reverse (`doc2author`).\n",
+    "\n",
+    "Below, we have also (this can be skipped for now):\n",
+    "* Increased the number of `passes` over the dataset (to improve the convergence of the optimization problem).\n",
+    "* Decreased the number of `iterations` over each document (related to the above).\n",
+    "* Specified the mini-batch size (`chunksize`) (primarily to speed up training).\n",
+    "* Turned off bound evaluation (`eval_every`) (as it takes a long time to compute).\n",
+    "* Turned on automatic learning of the `alpha` and `eta` priors (to improve the convergence of the optimization problem).\n",
+    "* Set the random state (`random_state`) of the random number generator (to make these experiments reproducible).\n",
+    "\n",
+    "We load the model, and train it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**FIXME:** why is autotuning turned on below?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 14min 10s, sys: 1min 12s, total: 15min 22s\n",
+      "Wall time: 14min 6s\n"
+     ]
+    }
+   ],
+   "source": [
+    "from gensim.models import AuthorTopicModel\n",
+    "%time model = AuthorTopicModel(corpus=corpus, num_topics=10, id2word=dictionary.id2token, \\\n",
+    "                author2doc=author2doc, chunksize=2000, passes=100, alpha='auto', eta='auto', \\\n",
+    "                eval_every=0, iterations=1, random_state=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before we explore the model, let's try to improve upon it. To do this, we will train several models with different random initializations, by giving different seeds for the random number generator (`random_state`). We evaluate the topic coherence of the model using the [top_topics](https://radimrehurek.com/gensim/models/ldamodel.html#gensim.models.ldamodel.LdaModel.top_topics) method, and pick the model with the highest topic coherence.\n",
```

Comment:
Actually c_v coherence is a better formula. it is easy to use as in https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/topic_coherence_tutorial.ipynb